### ISSUE TITLE
AttributeError: 'Keyword' object has no attribute 'teardown'

### ISSUE THREAD

**Issue by nicolasbillamboz-flowbird** [MarketSquare/robotframework-sherlock#91]
When i run sherlock on a folder containing a unique and very simple test:
```robotframework
*** Test Cases ***
My test
    Log    Hello test
``` 
I get the following error:
```
Sherlock analysis of Robot Framework code:

Using <my_project_root>/sandbox as source repository
Traceback (most recent call last):
  File "<my_project_root>/.venv/bin/sherlock", line 8, in <module>
    sys.exit(run_cli())
  File "<my_project_root>/.venv/lib/python3.8/site-packages/sherlock/__init__.py", line 19, in run_cli
    raise err
  File "<my_project_root>/.venv/lib/python3.8/site-packages/sherlock/__init__.py", line 9, in run_cli
    runner.run()
  File "<my_project_root>/.venv/lib/python3.8/site-packages/sherlock/core.py", line 37, in run
    suite.visit(code_visitor)
  File "<my_project_root>/.venv/lib/python3.8/site-packages/robot/model/testsuite.py", line 399, in visit
    visitor.visit_suite(self)
  File "<my_project_root>/.venv/lib/python3.8/site-packages/sherlock/visitor.py", line 98, in visit_suite
    suite.suites.visit(self)
  File "<my_project_root>/.venv/lib/python3.8/site-packages/robot/model/itemlist.py", line 102, in visit
    item.visit(visitor)    # type: ignore
  File "<my_project_root>/.venv/lib/python3.8/site-packages/robot/model/testsuite.py", line 399, in visit
    visitor.visit_suite(self)
  File "<my_project_root>/.venv/lib/python3.8/site-packages/sherlock/visitor.py", line 90, in visit_suite
    suite.tests.visit(self)
  File "<my_project_root>/.venv/lib/python3.8/site-packages/robot/model/itemlist.py", line 102, in visit
    item.visit(visitor)    # type: ignore
  File "<my_project_root>/.venv/lib/python3.8/site-packages/robot/model/testcase.py", line 186, in visit
    visitor.visit_test(self)
  File "<my_project_root>/.venv/lib/python3.8/site-packages/robot/model/visitor.py", line 157, in visit_test
    test.body.visit(self)
  File "<my_project_root>/.venv/lib/python3.8/site-packages/robot/model/itemlist.py", line 102, in visit
    item.visit(visitor)    # type: ignore
  File "<my_project_root>/.venv/lib/python3.8/site-packages/robot/model/keyword.py", line 65, in visit
    visitor.visit_keyword(self)
  File "<my_project_root>/.venv/lib/python3.8/site-packages/sherlock/visitor.py", line 115, in visit_keyword
    kw.teardown.visit(self)
AttributeError: 'Keyword' object has no attribute 'teardown'
Fatal exception occurred. You can create an issue at https://github.com/bhirsz/robotframework-sherlock/issues . Thanks!
```
**Environment**
- OS: Ubuntu 20.04
- Python: 3.8.10
- RobotFramework: 6.1
- robotframework-sherlock: 0.2.0

**Comment by nicolasbillamboz-flowbird** [comment]
@bhirsz you recently fixed and closed https://github.com/MarketSquare/robotframework-sherlock/issues/92, which I reported at the same time of this issue. This one is a blocker, but you haven't commented on it yet. Is it an oversight, or am I too impatient?

**Comment by bhirsz** [comment]
Don't worry, I'm keeping close eye on all issues but I just had time for one issue today and I picked randomly (mostly because it will be released in a few days in single release anyway). I should have more time tommorrow to take a look into this one (though it looks like something I fixed recently).


I also have finally access to real Robot Framework project (I was in company that didnt use RF for some time) so I will try to run some extra tests with Sherlock on bigger projects. It will be a lot easier to spot issues like this one.
